src/main/java/Katheryne.java

```diff
@@ -0,0 +1,91 @@
+import java.util.*;
```

Comment:
Should you list your imports explicitly for more clarity?

Reply:
+1 wildcard imports are not allowed according to Google's coding standard